invisible_cities/cities/components.py

```diff
@@ -898,24 +898,24 @@ def calculate_and_save_buffers(buffer_length    : float        ,
 
     saved_buffers = "buffers" if order_sensors is None else "ordered_buffers"
     max_subevt    =  max_time // buffer_length + 1
-    buffer_writer_    = sink(buffer_writer(h5out                  ,
-                                           run_number = run_number,
-                                           n_sens_eng = npmt      ,
-                                           n_sens_trk = nsipm     ,
-                                           length_eng = nsamp_pmt ,
-                                           length_trk = nsamp_sipm,
-                                           max_subevt = max_subevt),
+    buffer_writer_    = sink(buffer_writer(  h5out
+                                           , run_number = run_number
```

Comment:
I'm partial to aligning the commas directly under the opening bracket (and I'm pretty sure that is de-facto standard for this style), but I won't kick up a fuss if you leave it as it is.